taskcluster/fxci_config_taskgraph/transforms/firefoxci_artifact.py

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+from copy import deepcopy
+
+from taskgraph.transforms.base import TransformSequence
+
+from fxci_config_taskgraph.util.integration import (
+    STAGING_ROOT_URL,
+    find_tasks,
+    get_taskcluster_client,
+)
+
+transforms = TransformSequence()
+
+
+def get_artifact_urls(task_id: str, fetch_artifacts: list[str]) -> str:
+    queue = get_taskcluster_client("queue")
+    artifacts = queue.listLatestArtifacts(task_id)["artifacts"]
+
+    urls = [
+        queue.buildUrl("getLatestArtifact", task_id, artifact["name"]).replace(
+            "%2F", "/"
```

Comment:
My memory is a bit fuzzy, but I think it was because we need to extract the final path segment to get the file name to download to. I suppose I could have split the url on `%2F` instead, but it seemed cleaner to normalize it.